<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QMainWindow" name="ResultWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResultWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>triangle.png</normaloff>triangle.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="ParametersShowToolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>410</width>
      <height>261</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="ExtractedPointsPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>388</width>
       <height>175</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="label">
      <string>Извлеченные точки</string>
     </attribute>
     <widget class="QLabel" name="ColorPointsLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Цвет точек</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ColorPointsButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Triangulation2DPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>388</width>
       <height>175</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Триангуляция на плоскости</string>
     </attribute>
     <widget class="QLabel" name="ColorLinesLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Цвет линий</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ColorLinesButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Triangulation3DPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>408</width>
       <height>175</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Триангуляция в пространстве</string>
     </attribute>
     <widget class="QLabel" name="ColorTrianglesLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Цвет треугольников</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ColorTrianglesButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ColorSkeletonLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Цвет скелета</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ColorSkeletonButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="ShowModeGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>230</width>
        <height>170</height>
       </rect>
      </property>
      <property name="title">
       <string>Режим отображения</string>
      </property>
      <widget class="QRadioButton" name="ShowTrianglesRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>140</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Треугольники</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="ShowSkeletonRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>140</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Скелет</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ShowTrianglesAndSkeletonRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>210</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Треугольники и скелет</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="ShowMapOfHeightButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать карту высот</string>
    </property>
   </widget>
   <widget class="QLabel" name="ParametersShowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры отображения</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="infoTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>370</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ParametersTrianglesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры триангуляции</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSaveAs"/>
   <addaction name="actionShow"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>exit.png</normaloff>exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>show.png</normaloff>show.png</iconset>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
   <property name="toolTip">
    <string>Показать</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>save.png</normaloff>save.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="toolTip">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
